frontend/benefit/applicant/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+const { join } = require('path');
+
+module.exports = {
+  extends: ['auto'],
+  rules: {
+    '@typescript-eslint/no-floating-promises': 'off',
```

Comment:
hmm.. i was trying that, and then i realized that we have to use void everywhere and it looks....

Reply:
I think it's ok. `void` just marks that there is a promise that is intentionally not awaited:
https://stackoverflow.com/questions/67074848/angular-eslint-no-floating-promises-error-with-this-router-navigate

Forgetting await in some other place might cause hard-to-find bug. For example in events-helsinki there were a test that had:

`waitFor(() => expect(something));`

which always passed green, because `await` was forgotten from the beginning. When it was added the test failed, and there were an actual bug in the code that would have been catched. That false positive test were laying in the code from the beginning of the project.

Reply:
But if you find it very annoying, one compromise could be to use the no-floating-rule only in test files.